server/src/status.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class GameStatus(Enum):
+    pending = 0
```

Comment:
1) Names of enum must be in CAPS, 
2) Why not PENDING = 'pending'
3) class State(Enum), or GameState, (can be discussed)